src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -165,5 +217,48 @@ private void FindNodeForPath(object obj)
             }
             //visualizationManager.TagRenderPackageForPath(obj.ToString());
         }
+
+        /// <summary>
+        /// Method to account for the total number of items and the depth of a list in a list (in the WatchTree)
+        /// </summary>
+        /// 
+        public void CountNumberOfItems()
+        {
+            var listLevelAndItemCount = GetMaximumDepthAndItemNumber(this);
+            maxListLevel = listLevelAndItemCount.Item1;
+            NumberOfItems = listLevelAndItemCount.Item2;
+            IsCollection = maxListLevel > 0; 
+        }
+
+        private Tuple<int, int> GetMaximumDepthAndItemNumber(WatchViewModel wvm)
+        {
+            if (wvm.Children.Count == 0)
+            {
+                if (wvm.NodeLabel == "Empty List")
```

Comment:
Is there other way to check if it is an empty list or not? Suppose we localize "Empty List", this code will break